slack/web/async_slack_response.py

```diff
@@ -74,6 +74,10 @@ def __init__(
 
     def __str__(self):
         """Return the Response data if object is converted to a string."""
+        if isinstance(self.data, bytes):
```

Comment:
While SlackResponse has this validation, the same thing has been missing in `slack.web.AsyncSlackResponse` (legacy one).